src/features/documentLinkProvider.ts

```diff
@@ -6,6 +6,11 @@ import * as path from 'path'
 import * as vscode from 'vscode'
 import { OpenDocumentLinkCommand } from '../commands/openDocumentLink'
 import { getUriForLinkWithKnownExternalScheme } from '../util/links'
+import { AsciidocParser } from '../asciidocParser'
+import { similarArrayMatch } from '../similarArrayMatch'
+import { isSchemeBlacklisted } from '../linkSanitizer'
+
+const processor = require('@asciidoctor/core')()
```

Comment:
This is only used to locate regexes further below. But perhaps the regexes should just be duplicated since this is not really a public API. See https://github.com/asciidoctor/asciidoctor-vscode/pull/474/files#diff-013e1a7675f739b9f001eb11734529c1f730f4c81fa5cd146aa8fe4ae90bd2bcR94 for example.

Reply:
Good question... the regular expression for the include directive is not really complex, I think it would be safer to duplicate it.

Reply:
Just notice that we are also using `InlineLinkRx` and `InlineLinkMacroRx`, but still probably safer to duplicate it

Reply:
Got it, thanks.